src/views/Home.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="home">
+    <img alt="Vue logo" src="../assets/logo.png">
+    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  </div>
+</template>
+
+<script>
+// @ is an alias to /src
+import HelloWorld from '@/components/HelloWorld.vue'
```

Comment:
I am not able to see all the files. However, isn't the HelloWorld component deleted?

Reply:
Cleaned up and deleted the HelloWorld Component